napalm_eos/eos.py

```diff
@@ -1003,120 +1003,133 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = {}
 
+        # Placeholder for vrf arg
+        vrf = ''
+
+        # Right not iterating through vrfs is necessary
+        # show ipv6 route doesn't support vrf 'all'
+        if vrf == '':
+            vrfs = self.get_network_instances().keys()
+        else:
+            vrfs = [vrf]
+
```

Comment:
It is, but it makes the implementation of a vrf argument a simple change.  I can remove that if you want.